### PR TITLE
FIX: excludes deleted project from listing

### PR DESCRIPTION
**Pull request by vikramdevtron**
# Description

Excludes projects which are soft deleted, from listing and other places where they are visible.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Project Listing
- [x] Chart Deployed and App Create flow

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: internal/sql/repository/team/TeamRepository.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (impl TeamRepositoryImpl) Save(team *Team) error {
 
 func (impl TeamRepositoryImpl) FindAll() ([]Team, error) {
```
Comment (pghildiyal) [inline internal/sql/repository/team/TeamRepository.go:58]: change name of function it is FindAllActive

File: internal/sql/repository/team/TeamRepository.go [hunk 2]
```diff
@@ -92,15 +92,15 @@ func (repo TeamRepositoryImpl) FindTeamByAppName(appName string) (*Team, error)
 	team := &Team{}
```
Comment (pghildiyal) [inline internal/sql/repository/team/TeamRepository.go:92]: change name of function FindActiveTeamByAppName

File: internal/sql/repository/team/TeamRepository.go [hunk 3]
```diff
@@ -92,15 +92,15 @@ func (repo TeamRepositoryImpl) FindTeamByAppName(appName string) (*Team, error)
...
 	err := repo.dbConnection.Model(team).Column("team.*").
 		Join("inner join app a on a.team_id = team.id").Where("a.app_name = ?", appName).
-		Where("a.active = ?", true).Select()
+		Where("team.active = ?", true).Where("a.active = ?", true).Select()
 	return team, err
 }
 
 func (repo TeamRepositoryImpl) FindTeamByAppNameV2() ([]*TeamRbacObjects, error) {
```
Comment (pghildiyal) [inline internal/sql/repository/team/TeamRepository.go:99]: Fix the name, it is not getting team but rbacObjects also only active

Comment (vikramdevtron) [reply internal/sql/repository/team/TeamRepository.go:99]: removed, not in used

File: internal/sql/repository/team/TeamRepository.go (hunk 1 above)
Comment (vikramdevtron) [reply internal/sql/repository/team/TeamRepository.go:58]: done

File: api/restHandler/ClusterRestHandler.go [hunk 4]
```diff
@@ -215,7 +215,7 @@ func (impl ClusterRestHandlerImpl) FindAll(w http.ResponseWriter, r *http.Reques
 	token := r.Header.Get("token")
 	clusterList, err := impl.clusterService.FindAll()
 	if err != nil {
-		impl.logger.Errorw("service err, FindAll", "err", err)
+		impl.logger.Errorw("service err, FindAllActive", "err", err)
```
Comment (pghildiyal) [inline api/restHandler/ClusterRestHandler.go:218]: wll this change? i think this log message was update by mistake

Comment (vikramdevtron) [reply api/restHandler/ClusterRestHandler.go]: yes by mistake, fixed now

**Review by pghildiyal — APPROVED** [review]

**Pull request merged**